tools/epub_reader.py

```diff
@@ -0,0 +1,169 @@
+import zipfile
+from bs4 import BeautifulSoup
+
+#Open epub
+#Convert html to markdown
+#Prints mardown to .json files
+
+
+def _init_():
```

Comment:
Starting a function with a `_` give a function a special meaning in Python. It denotes some kind of metaprogramming or behind the scenes function that a library user wouldn't see. For this, I would recommend calling it `main`